frida_tools/tracer.py

```diff
@@ -573,6 +603,21 @@ def stop(self):
             self._script = None
 
     def _create_trace_script(self):
+        # Extract all init session files into a single string,
+        # substitute newlines with '\u000a'
+        self._init_session_code = ''
+        for path in self._init_session_paths:
+            # Append contents of <path> to <self._init_session_code>
+            fo = open(path, "rt")
+            line = fo.readlines()
+            fo.close()
```

Comment:
Better to use `with` here so there's no need to call `.close()`.

Reply:
Done (good point!).